src/app/auth/reducer.js

```diff
@@ -3,6 +3,7 @@ import * as types from './types';
 const initialState = {
   showModal: false,
   hasMetamask: true,
+  walletUnlocked: false,
```

Comment:
Needs to be set to false because in Firefox with Metamask `RECEIVE_ENABLE` is no returned when the wallet is locked.